be/src/olap/compaction.cpp

```diff
@@ -55,6 +55,25 @@ Status Compaction::execute_compact() {
     return st;
 }
 
+Status Compaction::samll_rowsets_compact() {
+    std::unique_lock<std::mutex> lock(_tablet->get_cumulative_compaction_lock(), std::try_to_lock);
+    if (!lock.owns_lock()) {
+        LOG(INFO) << "The tablet is under cumulative compaction. tablet=" << _tablet->full_name();
+        return Status::OLAPInternalError(OLAP_ERR_CE_TRY_CE_LOCK_ERROR);
+    }
+    _input_rowsets.clear();
+    int vertion_count = _tablet->version_count();
+    int64_t now = UnixMillis();
+    _tablet->pick_samll_verson_rowsets(&_input_rowsets);
+    if (_input_rowsets.size() >= 5) {
```

Comment:
Define a constant in place of the magic number

Reply:
ok